umh-core/pkg/fsm/redpanda/reconcile.go

```diff
@@ -91,7 +91,8 @@ func (r *RedpandaInstance) Reconcile(ctx context.Context, snapshot fsm.SystemSna
 	}
 
 	// Step 2: Detect external changes.
-	if err := r.reconcileExternalChanges(ctx, services, snapshot.Tick, start); err != nil {
+	err, reconciled = r.reconcileExternalChanges(ctx, services, snapshot.Tick, start)
```

Comment:
what is this change for?

Reply:
it will be overrriden by `err, reconciled = r.reconcileStateTransition(ctx, services, currentTime)`

do a reconciledExternal and reconciledStateTransition and then a || if you want to do it